### PR TITLE
performance_schema -> information_schema

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Stress that TiDB provides system tables in information_schema, not in performance_schema

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/6418
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
I've finished the self-check of this PR. PTAL. Thanks.

**Comment by morgo** [comment]
Looks good, but I'm sorry I can't review Chinese :-)

**Comment by shichun-0415** [comment]
> Looks good, but I'm sorry I can't review Chinese :-)

Get it. Please ignore the review request. Thanks :-)

File: statement-summary-tables.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ aliases: ['/docs-cn/dev/statement-summary-tables/','/docs-cn/dev/reference/perfo
 
 针对 SQL 性能相关的问题，MySQL 在 `performance_schema` 提供了 [statement summary tables](https://dev.mysql.com/doc/refman/5.6/en/statement-summary-tables.html)，用来监控和统计 SQL。例如其中的一张表 `events_statements_summary_by_digest`，提供了丰富的字段，包括延迟、执行次数、扫描行数、全表扫描次数等，有助于用户定位 SQL 问题。
 
-为此，从 4.0.0-rc.1 版本开始，TiDB 在 `information_schema` 中提供与 `events_statements_summary_by_digest` 功能相似的系统表：
+为此，从 4.0.0-rc.1 版本开始，TiDB 在 `information_schema` (_not_ `performance_schema`) 中提供与 `events_statements_summary_by_digest` 功能相似的系统表：
```
Comment (TomShawn) [inline statement-summary-tables.md:10]:
```suggestion
为此，从 4.0.0-rc.1 版本开始，TiDB 在 `information_schema` (_不是_ `performance_schema`) 中提供与 `events_statements_summary_by_digest` 功能相似的系统表：
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**